internal/query/org.go

```diff
@@ -202,8 +202,14 @@ func (q *Queries) SearchOrgs(ctx context.Context, queries *OrgSearchQueries) (or
 
 	query, scan := prepareOrgsQuery(ctx, q.client)
 	stmt, args, err := queries.toQuery(query).
-		Where(sq.Eq{
-			OrgColumnInstanceID.identifier(): authz.GetInstance(ctx).InstanceID(),
+		Where(sq.And{
+			sq.Eq{
+				OrgColumnInstanceID.identifier(): authz.GetInstance(ctx).InstanceID(),
+			},
+			sq.NotEq{
+				// Old removed state, orgs with this state should not show anymore
+				OrgColumnState.identifier(): domain_pkg.OrgStateRemoved,
+			},
```

Comment:
Hi @muhlemmer do you want me to change the version, the updated tests and remove the extra condition in this PR? 

I don't understand how changing the projection version will omit automatically any old row with org_state equals OrgStateRemoved, maybe it's a silly question but I'm still trying to grasp this eventstore/reduction magic :smile:

Reply:
Because the table is renamed, `setup` thinks it is missing. So a whole new table is created by the name  `projections.orgs1` and the events should be reduced into that one, using the updated reducers.

`projections.org` should still be there with reduced data corresponding the old reducing logic.

In a high availability like zitadel uses in production, this allows for a new version zitadel to be launched, create new projection(s) and populate them while an older version is still serving traffic using the old projection. Once the new projection is up-to-date production traffic can be redirected to the new version. If anything goes wrong, like a migration or unexpected errors in the new version, we can just make a switch back to the old version again.

So basically, we can operate multiple version of zitadel side-by-side from a database model perspective.

Reply:
Thanks again @muhlemmer for the explanation. I'll update the code removing the condition. I cant send the new code until next week due to lack of time.